splash-dao-offchain/src/routines/inflation/mod.rs

```diff
@@ -152,38 +286,81 @@ impl<'a, IB, PF, WP, VE, SF, PM, Backlog, Time, Actions, Bearer, Net>
         }
     }
 
-    async fn read_state(&self) -> RoutineState<Bearer>
+    async fn read_state(&mut self) -> RoutineState<Bearer>
```

Comment:
A bit counterintuitive. Why do we need mutable reference here if we only read

Reply:
Removed `mut`. I had it mutable because I call `try_pop()` on the `backlog` to check for latest voting order. But it's not strictly necessary since the `ResilientBacklog` trait doesn't need `mut` on this method: https://github.com/splashprotocol/splash-offchain-multiplatform/blob/ab932213a39c0a5dbcf1b7677b3db1f32775d3fd/spectrum-offchain/src/backlog.rs#L159